operators/multiclusterobservability/api/v1beta2/multiclusterobservability_types.go

```diff
@@ -16,6 +16,18 @@ type MultiClusterObservabilitySpec struct {
 	// Advanced configurations for observability
 	// +optional
 	AdvancedConfig *AdvancedConfig `json:"advanced,omitempty"`
+	// Size read path of your Observability instance
+	// +optional
+	// +kubebuilder:validation:Enum:=small;medium;large;xlarge;2xlarge;4xlarge;8xlarge
+	// +kubebuilder:default:="small"
+	ReadTShirtSize string `json:"readTShirtSize"`
```

Comment:
Just my 2 cents but personally I prefer to avoid `kubebuilder:default` and let the controller pick up the default value. It can be problematic when a tool tries to compare the applied manifest against the returned object. Also you won't be able to change the default value in the future if you need to.

Reply:
Ack! Will remove!